lib/src/jvmTest/kotlin/mathilda/RulesTest.kt

```diff
@@ -14,6 +14,10 @@ class RulesTest {
         val mathilda = builder(jsonStream).build()
 
         mathilda.rules.forEach { rule ->
+            if (rule.tests.isNotEmpty() && rule.tests[0].skip) {
+                return@forEach // Skip rules that are not applicable
+            }
```

Comment:
Could you please explain what this is for? Why do I want to skip tests? If there are tests that should be skipped, why don't just leave them out of the JSON? I don't understand this 😃

Reply:
In #4 I added a new `keep_params` rule and an example for Facebook. I think with the four existing rules now we should be able to migrate all existing rules to JSON. Please have a look and update your JSON. Thank you very much 😃